src/Commands/ProjectCreateCommand.php

```diff
@@ -89,6 +91,24 @@ protected function execute(InputInterface $input, OutputInterface $output): int
       }
     }
 
+    $project_disk_size = $input->getArgument('project_disk_size');
+
+    if (empty($project_disk_size)) {
+      $project_disk_size = $io->ask('Please provide a PlatformSH disk size for the project (default 8 GB)');
```

Comment:
I'd suggest the input value required, i.e. Please provide a PlatformSH disk size in megabytes for the project (default 8000MB)

Reply:
Good idea ! Have done that now